draft-tschofenig-rats-psa-token.md

```diff
@@ -212,13 +222,18 @@ Note that the CDDL label used to be called arm_psa_partition_id.
 ## Target Identification Claims
 
 ### Instance ID
+{: #sec-instance-id-claim}
 
-The Instance ID claim represents the unique identifier of the device instance.
+The Instance ID claim represents the unique identifier of the device.
 It is a 32 bytes hash of the public key corresponding to the Initial
```

Comment:
The current PSA Security model has backed off from this strict definition of the Instance ID. It now just says it 'identifies the IAK'.

Reply:
We'll also want to revise it to also cover group unique keys as an alternative

Reply:
Unless there is a separate, distinct EAT claim for that?

Reply:
@SimonFrost-Arm : makes sense, I will remove the details about the derivation.

@adrianlshaw : can you suggest some text?

Reply:
> Unless there is a separate, distinct EAT claim for that?

There is some discussion ongoing about defining a separate "group ueid" coming from the FIDO side.

Reply:
On a re-read, I think 'identifies the IAK' should be fine for both cases.

Reply:
done in 58ba0c6